src/web_ui/__init__.py

```diff
@@ -2,16 +2,18 @@
 import logging
 
 from flask import Flask, render_template, request
-from node_mgmt import NetworkEnv, EnvScanner, get_ssh_fingerprint
-from node_mgmt.commands import (
+
+from src.node_mgmt import NetworkEnv, EnvScanner, get_ssh_fingerprint, Node
```

Comment:
Doesn't seem right that we're prepending `src` to all module imports. The `src` directory is the source root and modules should be "under" it.

Reply:
I'm not sure how we're running this code tbh, I'll revert these changes

Reply:
Yeah there's a setup.py which installs these as modules